libs/service-portal/settings/access-control/src/screens/GrantAccess/GrantAccess.tsx

```diff
@@ -216,7 +218,7 @@ function GrantAccess() {
                   onChange={(value) => {
                     requestDelegation(value)
                   }}
-                  size="md"
+                  size="xs"
```

Comment:
I thought we were happy with "md" because it's just one "primary" input on the page? :D

Reply:
whops - I'm so used to changing it all to small and xs 😆  I'll change it back!

Reply:
done